broadway_api/flags.py

```diff
@@ -0,0 +1,124 @@
+from flagset import Flag
+
+app_flags = {
+    "debug": Flag(
+        bool,
+        default=False,
+        cmdline_name=["-d", "--debug"],
+        env_name="BROADWAY_DEBUG",
+        config_name="debug",
+        help="enable debug mode",
+    ),
+    "token": Flag(
+        str,
+        required=True,
+        cmdline_name="--token",
+        env_name="BROADWAY_TOKEN",
+        config_name="token",
+        help="access token for communication between workers and api",
+    ),
+    "heartbeat_interval": Flag(
+        int,
+        default=10,
+        cmdline_name="--heartbeat-interval",
+        config_name="heartbeat_interval",
+        help="heartbeat interval",
+    ),
+    "course_config": Flag(
+        str,
+        cmdline_name="--course-config",
+        config_name="course_config",
+        help="optional course config file",
```

Comment:
Might be worth mentioning what would happen if this is not provided? That the existing configuration will be used. If provided, existing configuration is completely overwritten.